packages/playwright-ct-react/register.mjs

```diff
@@ -30,6 +30,10 @@ function render(component) {
     if (typeof child === 'string')
       return child;
     return render(child);
+  }).filter(child => {
```

Comment:
Is this react specific, or should we copy this to other ct packages?

Reply:
Other frameworks use slots that operate fragments containing text nodes.